about.html

```diff
@@ -0,0 +1,118 @@
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            <ul class="menu">
+                <li><a href="./index.html" class="nav-link">Home</a></li>
+                <li><a href="./#program" class="nav-link">Program</a></li>
+                <li><a href="" class="nav-link">Join</a></li>
+                <li><a href="" class="nav-link">Sponsor</a></li>
+                <li><a href="" class="nav-link">News</a></li>
+                <li><a href="" class="nav-link">CC Campaign</a></li>
+        </nav>
+        <main class="headline">
+            <h1 class="title">Buid The Next Big Thing On Web3</h1>
+            <h3 class="sub-title">
+                Our World has changed. - Hack your way to the top
+            </h3>
+            <p class="title-description">
+                WEB3 is here to help reimagine your world. Write, test and deploy scalable decentralized applications on the most developer-friendly blockchain using the languages you already know – even Javascript.
+            </p>
+            <span class="contact-us">
+                Please contact us per Email for any further questions about WEB3 Hackathon 2022!
+            </span>
+            <p class="contact-email">
+                web3hackathon2022@techblaze.org
+            </p>
+        </main>
+    </div>
+    </section>
+    <section class="hackathon-logo" id="hackathon-logo">
+        <h2 class="hackathon-logo-title">WEB3 Hackathon 2022 Logo</h2>
+        <hr>
+        <p class="logo-design">
+            The logo for the 2022 WEB3 HACKATHON was decided through a unanimous community vote after 
+            different designs have been submitted by talented designers in the community. The design from
+            Valentine Blaze emerged the winner, accumulating the highest votes from the pulll.
+        </p>
+        <div class="hackathon-logo-img">
+            <img src="./assets/images/Web3-Hackathon.png" alt="Hackathon logo" class="hackathon-logo-image">
+        </div>
+    </section>
+    <section  id="past-events">
+        <div class="past-event-section">
+            <h2 class="past-event-title">See the past Web3 Hackathon Events</h2>
+            <hr>
+            <p class="past-info">
+                Take a look at the last two Web3 Hackathon events which took place in Lagos-Nigeria and London
+            </p>
+            <div class="past-event-images">
+                <div class="event-1">
+                    <div class="event-img-overlay">
+                        <!-- <img src="" alt="Past Event 1"> -->
+                        <span class="event-year"> 2021</span>
+                        <span class="event-detail"> Web3 Hackathon 2021 Lagos-Nigeria</span>
+                    </div>
+                </div>
+                <div class="event-2">
+                    <div class="event-img-overlay">
+                        <!-- <img src="" alt="Past event 2"> -->
```

Comment:
- [ ] Kindly remove commented pieces of code for better readability and for a more professional code 💯